### PR TITLE
Made filter configuration file. Added set methods.

### LINKED ISSUE #5: Create FilterConfigurationBuilder class to pass group of arguments
class FilterConfigurationBuilder {
  final Map<String, dynamic> _params = {};

  void setNSNumber(String key, num value) {
    _params[key] = value;
  }

  void setFloat(String key, num value) {

  }

  void setNSData(String key, Uint8List data) {}

  void setNSDataAsset(String key, String asset) {}

  void setNSDataFile(String key, File file) {}

  Map<String, dynamic> get params => Map.of(_params);
}

https://github.com/MasayukiSuda/GPUVideo-android/tree/master/gpuv/src/main/java/com/daasuu/gpuv/egl/filter
https://github.com/cats-oss/android-gpuimage/tree/master/library/src/main/java/jp/co/cyberagent/android/gpuimage/filter
https://cifilter.io/

### PR DESCRIPTION
**Pull request by EgorEko**
Closes #5

### CODE REVIEW

File: lib/factory/filter_configuration_builder.dart [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+  void setFloat(String key, num value) {
+    _params[key] = value;
+  }
+
+  void setFloatList(String key, List<num> value) {
+    _params[key] = value;
+  }
+
+  void setInt(String key, num value) {
```
Comment (nikolaydymura) [inline lib/factory/filter_configuration_builder.dart:52]: Use int here

File: lib/factory/filter_configuration_builder.dart [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    _params[key] = value;
+  }
+
+  //TODO: questionable
+  void setNSString(String key, String value) {
+    _params[key] = value;
+  }
+
+  //TODO: questionable
+  void setNSValue(String key, dynamic value) {
```
Comment (nikolaydymura) [inline lib/factory/filter_configuration_builder.dart:84]: Change to List<double>

File: lib/factory/filter_configuration_builder.dart [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    _params[key] = value;
+  }
+
+  void setPoint(String key, Point value) {
```
Comment (nikolaydymura) [inline lib/factory/filter_configuration_builder.dart:88]: Change to Point<num>

**Review by nikolaydymura — CHANGES_REQUESTED** [review]

File: lib/factory/filter_configuration_builder.dart [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+  void setBitmap(String key, Uint8List data) {
+    _params[key] = data;
+  }
+
+  void setBitmapAsset(String key, String asset) {}
+
+  void setBitmapFile(String key, File file) {}
+
+  void setBool(String key, num value) {
```
[the remark this answers is not in the document]
Comment (EgorEko) [reply lib/factory/filter_configuration_builder.dart:27]: Changed to bool

File: lib/factory/filter_configuration_builder.dart [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+    _params[key] = value;
+  }
+
+  void setCIColor(String key, Color value) {
+    _params[key] = value;
+  }
+
+  //TODO: questionable
+  void setCIVector(String key, CIVector value) {
```
[the remark this answers is not in the document]
Comment (EgorEko) [reply lib/factory/filter_configuration_builder.dart:36]: Changed to List<double>

File: lib/factory/filter_configuration_builder.dart (hunk 2 above)
Comment (EgorEko) [reply lib/factory/filter_configuration_builder.dart:84]: Changed to List<double>

File: lib/factory/filter_configuration_builder.dart (hunk 1 above)
Comment (EgorEko) [reply lib/factory/filter_configuration_builder.dart]: Changed to int

**Review by nikolaydymura — APPROVED** [review]

**Pull request merged**